pyproject.toml

```diff
@@ -0,0 +1,76 @@
+# Copyright Quantinuum
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "quantinuum_qircheck"
+description = "Package to check QIR for compatibility with quantinuum devices"
+readme = "README.md"
+requires-python = ">=3.10, <3.13"
+license = { file = "LICENSE" }
+authors = [{ name = "Quantinuum" }]
+maintainers = [
+  { name = "TKET development team", email = "tket-support@quantinuum.com" },
+]
+
+classifiers = [
+        "Environment :: Console",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: MacOS :: MacOS X",
+        "Operating System :: POSIX :: Linux",
+        "Operating System :: Microsoft :: Windows",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering",
+]
+dynamic = ["version"]
+dependencies = [
+  "pyqir == 0.10.6",
+]
+
+[project.optional-dependencies]
+test = ["pytest"]
+
+[project.urls]
+Repository = "https://github.com/CQCL/quantinuum-qircheck"
+Tracker = "https://github.com/CQCL/quantinuum-qircheck/issues"
+
+[tool.setuptools.packages.find]
+where = ["."]
+
+[tool.setuptools.package-data]
+"pytket.phir" = ["py.typed"]
+
+[tool.pytest.ini_options]
+addopts = "-s -vv"
+pythonpath = ["."]
+log_cli = true
+log_cli_level = "INFO"
+log_level = "DEBUG"
+filterwarnings = ["ignore:::lark.s*"]
```

Comment:
Not needed.

Reply:
done in [clean up project.toml](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/bf84776866cf85089ffdd3a88269f6539120f4d3)
[bf84776](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/bf84776866cf85089ffdd3a88269f6539120f4d3)